Feisty/GameDetails/GameDetailsViewController.swift

```diff
@@ -49,6 +51,15 @@ class GameDetailsViewController: UIViewController {
 
   }
   
+  private func setUpGameCardDropShadow() {
+    
+    gameCard.layer.shadowColor = UIColor.black.cgColor
```

Comment:
Can't you use the shadow extension here too ?

Reply:
I want to, I only thought about doing the extension much later on. So once that gets into development I can go through the entire project and remove much unneeded code